etc/profile-m-z/steam.profile

```diff
@@ -35,6 +35,7 @@ noblacklist ${HOME}/.local/share/vpltd
 noblacklist ${HOME}/.local/share/vulkan
 noblacklist ${HOME}/.mbwarband
 noblacklist ${HOME}/.paradoxinteractive
+noblacklist ${HOME}/.prey
```

Comment:
Never blacklisted.

Reply:
@rusty-snake [on Mar 11](https://github.com/netblue30/firejail/pull/5029#discussion_r824904789):

> Never blacklisted.

Fixed on commit d34390733 ("disable-programs.inc: add ~/.prey", 2022-03-11).

Reply:
Sorry, I didn't realize there's an explicit blacklist for individual paths.